sdk/core/azure-core-tracing-opentelemetry/pom.xml

```diff
@@ -126,20 +126,11 @@
       <activation>
         <jdk>[9,)</jdk>
```

Comment:
We should standardize using 11 and above for these profiles to be consistent with all modules.

Reply:
I've actually wondered if this should be Java 9 or Java 11, I know our CI uses Java 11 but technically this is for Java 9+. Given that I've actually been thinking this should switch on in Java 9.

Reply:
There is a pre-existing tracking item for this: https://github.com/Azure/azure-sdk-for-java/issues/19273